cmd/olympus/actions/actions_test.go

```diff
@@ -8,34 +8,25 @@ import (
 	"github.com/gobuffalo/gocraft-work-adapter"
 	"github.com/gobuffalo/suite"
 	"github.com/gocraft/work"
-	"github.com/gomods/athens/pkg/config/env"
+	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/eventlog"
-	"github.com/gomods/athens/pkg/eventlog/mongo"
 	"github.com/gomods/athens/pkg/payloads"
-	"github.com/gomods/athens/pkg/storage/mem"
+)
+
+const (
+	testConfigFile = "../../../config.test.toml"
 )
 
 type ActionSuite struct {
 	*suite.Action
 }
 
 func Test_ActionSuite(t *testing.T) {
-	stg, err := mem.NewStorage()
+	conf := config.GetConfLogErr(testConfigFile, t)
```

Comment:
Not a blocker but don't you think that just
```
c, err := GetConf(path)
if err != nil {
	t.Fatalf("Unable to parse config file: %s", err.Error())
}
```
instead of this func would be okay as well?

Reply:
absolutely. I just ended up writing that so many times that I wanted it in a reusable function :)